lapce-proxy/src/plugin/wasi.rs

```diff
@@ -233,6 +233,12 @@ pub fn find_all_volts() -> Vec<VoltMetadata> {
             d.read_dir().ok().map(|dir| {
                 dir.filter_map(|result| {
                     let entry = result.ok()?;
+                    if entry.file_type().ok()?.is_file() {
+                        return None;
+                    }
+                    if entry.file_name().to_str()?.starts_with('.') {
+                        return None;
+                    }
```

Comment:
```suggestion
                    let entry = entry.metadata()?;
                    if !entry.is_dir() || entry.file_name().to_str()?.starts_with('.') {
                        return None;
                    }
```

Reply:
metadata does not contain file_name. I can remove the variable init if you prefer, seeing as it's only used on is_dir

Reply:
you committed outdated suggestion

Reply:
🤦 Sorry about that

Reply:
could probably use `if metadata.is_file()` to not break symlinks